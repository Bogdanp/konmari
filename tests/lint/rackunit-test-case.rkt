#lang racket/base

(require rackunit)

(define expected 42)
(define bar 1)

(test-suite
 "a"

 (test-case "b"
   (define foo 42)
   (check-equal? foo expected))

 (test-case "c"
   (define foo 43)
   (check-not-equal? foo expected)))
