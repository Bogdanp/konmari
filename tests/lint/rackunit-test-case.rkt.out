/rackunit-test-case.rkt:6:9:warning:identifier 'bar' is never used
